js/apps/events-page/directives/datepickers.js

```diff
@@ -20,22 +20,41 @@
 						innerScope.userEndDate = innerScope.userEndDate || getEndDateLocaleString();
 
 						$timeout(() => {
-							if ($window.moment(innerScope.userStartDate).isSameOrBefore(innerScope.userEndDate)) {
-								innerScope.areDatesInvalid = false;
+							innerScope.areDatesInvalid = isDateRangeInvalid();
+
+							if (!innerScope.areDatesInvalid) {
 								innerScope.filterByDate();
-							} else {
-								innerScope.areDatesInvalid = true;
 							}
 						});
 					});
 				}
 			};
 
+			innerScope.updateEndDate = () => {
+				if (isDateRangeInvalid()) {
+					innerScope.userEndDate = $window
+						.moment(innerScope.userStartDate)
+						.add(1, 'd')
+						.format('MMMM DD, YYYY');
+				}
+
+				$timeout(() => {
+					innerScope.filterByDate();
+				});
+			};
+
 			innerScope.openFlatpickr = (flatpickrElementId) => {
 				const flatpickr = document.querySelector('#' + flatpickrElementId)._flatpickr; // eslint-disable-line no-underscore-dangle
 				flatpickr.open();
 			};
-      
+
+			const isDateRangeInvalid = () => {
+				if ($window.moment(innerScope.userStartDate).isSameOrBefore(innerScope.userEndDate)) {
```

Comment:
you can simplify this method to

```js
const isDateRangeInvalid = () => !$window.moment(innerScope.userStartDate).isSameOrBefore(innerScope.userEndDate);
```

Reply:
Good catch. I have simplified that method now.